src/prefect/utilities/notifications/notifications.py

```diff
@@ -131,24 +131,37 @@ def slack_message_formatter(
     if value is not None:
         fields.append({"title": "Message", "value": value, "short": False})
 
-    data = {
-        "attachments": [
-            {
-                "fallback": "State change notification",
-                "color": state.color,
-                "author_name": "Prefect",
-                "author_link": "https://www.prefect.io/",
-                "author_icon": "https://emoji.slack-edge.com/TAN3D79AL/prefect/2497370f58500a5a.png",
-                "title": type(state).__name__,
-                "fields": fields,
-                #                "title_link": "https://www.prefect.io/",
-                "text": "{0} is now in a {1} state".format(
-                    tracked_obj.name, type(state).__name__
-                ),
-                "footer": "Prefect notification",
-            }
-        ]
+    notification_payload = {
+        "fallback": "State change notification",
+        "color": state.color,
+        "author_name": "Prefect",
+        "author_link": "https://www.prefect.io/",
+        "author_icon": "https://emoji.slack-edge.com/TAN3D79AL/prefect/2497370f58500a5a.png",
+        "title": type(state).__name__,
+        "fields": fields,
+        #                "title_link": "https://www.prefect.io/",
```

Comment:
nit pick: might as well delete this line

Reply:
I was thinking that too but looks like it was commented out prior to this PR and my thought went with that it's there to serve as an example